### PR TITLE
CVP-1871 subtasks: Remove numbered rules from skip_list

### PR DESCRIPTION
**Pull request by LikhithaEda**

### CODE REVIEW

File: roles/collect_modified_images/tasks/parse_image_names.yml [hunk 1]
```diff
@@ -21,4 +21,4 @@
 - name: "parse and return image name when timestamp is after start_time"
   set_fact:
     pulled_images: "{{ pulled_images + [image] }}"
-  when: "{{ timestamp > deployment_start_time }}"
+  when: timestamp > deployment_start_time
```
Comment (wheelerlaw) [inline roles/collect_modified_images/tasks/parse_image_names.yml:24]: This doesn't seem right, does the interpolation not happen when the curly brackets are removed? @samvarankashyap

Comment (samvarankashyap) [reply roles/collect_modified_images/tasks/parse_image_names.yml:24]: Inside when condition, we need not have the curly braces 
variables can be accessed directly afaik 
@wheelerlaw

**Comment by 14rcole** [comment]
This all looks good to me.  Did we disable the linting for the line to long error in one of the hotfixes yesterday? @samvarankashyap @sonam1412

File: roles/collect_modified_images/tasks/parse_image_names.yml (hunk 1 above)
Comment (wheelerlaw) [reply roles/collect_modified_images/tasks/parse_image_names.yml:24]: I see, thank you

**Review by wheelerlaw — APPROVED** [review]

**Review by wheelerlaw — APPROVED** [review]

**Review by wheelerlaw — APPROVED** [review]

**Review by 14rcole — APPROVED** [review]

**Comment by sonam1412** [comment]
> long error in one of the hotfixes yesterday

yes

**Pull request merged**